README.md

```diff
@@ -292,6 +292,32 @@ We can see that the word `"actual"` in this case is referring to the
 **string** (`"39"`). The test will interpret the value as a string due to the
 quotation marks wrapping it.
 
+## Optional Mocha Configuration
+
+In this lab, we only had three tests to pass, but as you continue through the
+curriculum you will encounter labs with many more tests. You can imagine that
+the test output could get very long, making it more difficult to focus in on how
+to fix a particular error.
+
+To help with this issue, there is some very simple setup you can put in place in
+Mocha that will cause the tests to stop as soon as the first failing test is
+encountered.
+
+To implement this, first create a new file, `mocha.opts`, in the `test`
+directory:
+
+```sh
+touch test/mocha.opts
+```
+
+Then open the file in your text editor and add the following:
+
+```text
+--bail
+```
```

Comment:
Adding `--bail` at the end of the script in the `package.json` file also does the trick and might be easier for students to manage. That like would look like this:

```
"test": "mocha --timeout 5000 --reporter 'json' > .results.json & mocha --bail"
```

(Aside: I forgot that we changed from using mocha-multi for some of the prework labs, so this script is a bit different than what they'd see in Phase 1... it technically runs mocha twice, the first time to generate the json report and the second time to show the test output in the terminal)

I tried this out on a Phase 1 lab using `learn test` and everything seemed to work ok, so it looks like this is a nice workflow to give students!